### PR TITLE
readme instructions elaboration updated

### PR DESCRIPTION
**Pull request by Shoaib7889**
#### Just a little bit more elaborated the instructions of the readme file where I found it difficult to understand.
Fixing issue #4909

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,20 +10,19 @@
 
 ### Contributing
 
-OpenEMR is a leader in healthcare open source software and comprises a large and diverse community of software developers, medical providers and educators with a very healthy mix of both volunteers and professionals.  [Join us and learn how to start contributing today!](https://open-emr.org/wiki/index.php/FAQ#How_do_I_begin_to_volunteer_for_the_OpenEMR_project.3F)
+OpenEMR is a leader in healthcare open source software and comprises a large and diverse community of software developers, medical providers and educators with a very healthy mix of both volunteers and professionals. [Join us and learn how to start contributing today!](https://open-emr.org/wiki/index.php/FAQ#How_do_I_begin_to_volunteer_for_the_OpenEMR_project.3F)
 
-> Already comfortable with git? Check out [CONTRIBUTING.md](CONTRIBUTING.md) for quick setup instructions
+> Already comfortable with git? Check out [CONTRIBUTING.md](CONTRIBUTING.md) for quick setup instructions and let's start contributing.
```
Comment (adunsulag) [inline README.md:15]: I'm sorry but this sentence seems even more confusing to me as a native English speaker.  Perhaps changing this to be:
```Already comfortable with git? Check out [CONTRIBUTING.md](CONTRIBUTING.md) for quick setup instructions and requirements for contributing to OpenEMR```

**Review by adunsulag — CHANGES_REQUESTED** [review]
Thanks for trying to help improve the documentation @Shoaib7889. I had a couple of questions and suggestions for improvement as a native English Speaker.  

Also, we've recently implemented a new policy requiring Pull requirements to have an attached issue.  So once you've gone through my suggestions, please open up a new issue and then reference it here in your PR.

File: README.md (hunk 1 above)
Comment (Shoaib7889) [reply README.md:15]: Cool, Now it seems more elaborative, 
Let me change it to the suggested one or some more meaningful.

File: README.md [hunk 2]
```diff
@@ -10,20 +10,19 @@
...
 Community and Professional support can be found [here](https://open-emr.org/wiki/index.php/OpenEMR_Support_Guide).
 
-Extensive documentation and forums can be found on the [OpenEMR website](https://open-emr.org).
+Extensive documentation and forums can be found on the [OpenEMR website](https://open-emr.org) that can help you to become more familiar about the project.
 
 ### Reporting Issues and Bugs
 
-Report these on the [Issue Tracker](https://github.com/openemr/openemr/issues). If you are unsure if it is an issue/bug, then always feel free to use the [Forum](https://community.open-emr.org/) and [Chat](https://www.open-emr.org/chat/).
-
+Report these on the [Issue Tracker](https://github.com/openemr/openemr/issues). If you are unsure if it is an issue/bug, then always feel free to use the [Forum](https://community.open-emr.org/) and [Chat](https://www.open-emr.org/chat/) to discuss about the it.
```
[the remark this answers is not in the document]
Comment (Shoaib7889) [reply README.md:25]: Mistakenly I wrote 'the it', 
It should be 'to discuss about it' or something more meaningful.

File: README.md [hunk 3]
```diff
@@ -43,7 +42,8 @@ Check out [FHIR_README.md](FHIR_README.md)
 
 ### For Developers
 
-If using OpenEMR directly from the code repository, then the following commands will build OpenEMR (Node.js version 12.* or 14.* is required) :
+If using OpenEMR directly from the code repository, then the following commands will build OpenEMR (Node.js version 12._ or 14._ is required) :
```
[the remark this answers is not in the document]
Comment (Shoaib7889) [reply README.md:45]: Hmmm, 
Sorry, I think this is due to prettier linting issues. Let me fix it.

**Comment by bradymiller** [comment]
already brought in via subsequent PR: https://github.com/openemr/openemr/pull/4910

**Pull request closed without merging**